easybuild/easyconfigs/l/libgpuarray/libgpuarray-0.7.6-fosscuda-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'CMakeMake'
+
+name = 'libgpuarray'
+version = '0.7.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeplearning.net/software/libgpuarray/'
+description = "Library to manipulate tensors on the GPU."
+
+toolchain = {'name': 'fosscuda', 'version': '2020a'}
+
+source_urls = ['https://github.com/Theano/libgpuarray/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['887b6433a30282cb002117da89b05812c770fd9469f93950ff3866ddd02bfc64']
+
+builddependencies = [('CMake', '3.16.4')]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('Mako', '1.1.2'),
+    ('NCCL', '2.8.3', '-CUDA-11.0.2', True),
```

Comment:
I will try to get fix into framework that allows for `%(cudaver)s` template here instead of specifying the number again.

Reply:
@Micket any progress on the framework fix?

Reply:
https://github.com/easybuilders/easybuild-framework/pull/3541

Reply:
Good to go i think (even if @boegel wants to adjust the way the template is populated, this should still work)